### DISCUSSION TITLE
Is Android 5/6 support were brought back?

### DISCUSSION THREAD

**Discussion by leap0x7b** [termux/termux-app#2752] [General]
When I want to install the latest nightly build of Termux on GitHub Actions, I noticed that there's a new branch called [`android-5-reborn`](https://github.com/termux/termux-app/tree/android-5-reborn). Does that mean that Android 5/6 support is going to be brought back?

**Comment by ghost** [comment]
It will only backport important security fixes and updates, including updating repository lists from android 5 bootstraps. packages will still be in frozen state from 2 years behind however.
https://github.com/termux/termux-app/issues/2651
https://github.com/termux/termux-app/pull/2740

**Comment by sylirre** [comment]
https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6
